### PR TITLE
Updates 2018Aug

### LINKED ISSUE #152: Social media metadata
There are several metadata things that can be added to the site to increase compatibility/visibility on social media and search platforms. I have a partial list elsewhere that I'll update this issue with; this will be a Fall 2018 task

### LINKED ISSUE #155: Add "subscribe" buttons to Announcements page
- RSS
- Email

### PR DESCRIPTION
**Pull request by ct-martin**
* Config updates for HTTPS & more control over excerpts
* Improve Project page template, index (structure completed for #153, pending more content). Update TigerOS page to override excerpt
* Add Teleirc page & bump number on TigerOS page
* Fixes #155 - Links to mailing list and rss feed on announcements & talks lists (inc. home page)
* Fixes #152 - Social media SEO metadata

### CODE REVIEW

File: projects/_posts/2018-01-01-teleirc.md [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+layout: project
+title: Teleirc
+permalink: /projects/teleirc
+authors:
+- Justin Flory (@jwf)
+- Seth Hendrick (@xforever1313)
+- Mark Repka (@repkam09)
+- Nath Levesque (@thenaterhood)
+- More!
+excerpt: >
```
Comment (Tjzabel) [inline projects/_posts/2018-01-01-teleirc.md]: I'm confused as to how the excerpt interacts with the rest of the page. This text is repeated in the About section

Comment (ct-martin) [reply projects/_posts/2018-01-01-teleirc.md]: It's a technical thing

Comment (ct-martin) [reply projects/_posts/2018-01-01-teleirc.md]: (Overrides a value that Jekyll sort of guesses at, but I wanted it to be cleaner)

File: projects/_posts/2018-01-01-teleirc.md [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+  <p>Teleirc bridges communication between a Telegram group and an IRC channel. The project is a lightweight NodeJS application.</p>
+  <p>This bot was originally written for RITlug and our own Telegram group and IRC channel. Today, it is used by various communities other than just RITlug.</p>
```
[the remark this answers is not in the document]
Comment (ct-martin) [reply projects/_posts/2018-01-01-teleirc.md]: Sure. I had copied the Teleirc readme, so I didn't really focus on the content so much

File: projects/_posts/2018-01-01-teleirc.md (hunk 1 above)
Comment (ct-martin) [reply projects/_posts/2018-01-01-teleirc.md]: For reference, everything in the triple-dash (---) section at the top of any page in a Jekyll site is metadata Jekyll can use to do things

**Pull request merged**